packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -151,7 +155,7 @@ export const Headline = styled.h3`
   }}
 
   color: ${C_EBON};
-  ${({ service }) => getSerifBold(service)}
+  ${({ service }) => getSerifMedium(service)}
```

Comment:
Why are we changing this from `getSerifBold` to `getSerifMedium`?

Reply:
As per UX said:
>The headlines on promos have always been Reith Serif Medium (fallback is sans bold for services not getting Reith).